doc/governance.rst

```diff
@@ -139,6 +141,76 @@ Team leads are selected by the executive council, generally based on
 the recommendation by team members and outgoing lead.  Periodic
 rotation of lead responsibility among team members is encouraged.
 
+Current Component Leads
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Component
+     - Description
+   * - Adam King
+     - Cephadm
+     - Ceph's native orchestrator (install, upgrade...)
+   * - Venky Shankar
+     - CephFS
+     - Shared filesystem
+   * - Matan Breizman
+     - Crimson
+     - Next generation implementation of the OSD
+   * - Afreen Misbah
+     - Dashboard
+     - Ceph's graphical user interface
+   * - Radoslaw Zarzynski
+     - RADOS
+     - Ceph's low level storage foundation
+   * - Ilya Dryomov
+     - RBD
+     - Block storage for containers and VMs
+   * - Adam Emerson, Eric Ivancich
+     - RGW
+     - S3/Swift compatible object storage
+   * - Yingxin Cheng
+     - Seastore
+     - Next generation storage underlying the OSD
+
+
+General Leads
+=============
+
+These are general areas rather than specific subsystems within Ceph.
+Each is unique.
+
+Documentation
+-------------
+
+Responsibilities
+^^^^^^^^^^^^^^^^
+* Update the documentation to describe new capabilities.
+* Solicit user feedback and respond to it.
+* Maintain communications channels so that members of the upstream community have confidence that their documentation concerns are recognized and addressed.
+* Correct the structure of sentences as needed.
+* Ensure that the .rst files and the .md files that constitute the documentation are properly formatted and that Sphinx renders them as intended.
+* Ensure that correct information is backported from the main/ branch to the documentation release branches.
```

Comment:
Suggest
>from the ``main`` branch
instead of
>from the main/ branch

Reply:
Also

* Enhance the ability of the community to contribute to documentation and facilitate timely PR reviews
* Facilitate the ceph.io blog site

Reply:
I've added this.